MAUI/Linear-Gauge/getting-started.md

```diff
@@ -249,4 +249,6 @@ The following code example gives you the complete code of above configurations.
 
 ![.NET MAUI Linear Gauge](images/getting-started/maui-linear-gauge.PNG)
 
-N> View [`sample`](https://github.com/SyncfusionExamples/MAUI-Linear-Gauge-Getting-started-) in GitHub
```

Comment:
Why is this deleted (no need to midify this)..

Reply:
Done

Reply:
Recheck..